examples/rpc/partner-plugin/docs/example-messages/accommodation_search.json

```diff
@@ -0,0 +1,154 @@
+{
+    "header": {
+        "base_header": {
+            "end_user_wallet_address": "enim tempor ad aliqua",
+            "version": {
+                "major": -333443382,
+                "minor": -2064758523,
+                "patch": -1139049495
+            }
+        }
+    },
+    "metadata": {
+        "external_session_id": "123session"
+    },
+    "queries": [
+        {
+            "query_id": 1,
+            "search_parameters_accommodation": {
+                "location_geo_tree": {
+                    "country": 13,
+                    "region": "Balearic Islands",
+                    "city_or_resort": "Mallorca"
+                }
+            },
+            "travel_period": {
+                "start_date": {
+                    "year": 2024,
+                    "month": 11,
+                    "day": 15
+                },
+                "end_date": {
+                    "year": 2024,
+                    "month": 11,
+                    "day": 18
+                },
+                "flexible": true,
+                "los_from_nights": 0,
+                "los_to_nights": 0,
+                "dow": {
+                    "dow": 0
+                }
+            },
+            "travellers": [
+                {
+                    "traveller_id": 1785472694,
+                    "type": "TRAVELLER_TYPE_CHILD",
+                    "birthdate": {
+                        "year": 2011,
+                        "month": 2,
+                        "day": 16
+                    },
+                    "nationality": 142
+                },
+                {
+                    "traveller_id": 1156774514,
+                    "type": 4,
+                    "birthdate": {
+                        "year": 2006,
+                        "month": 4,
+                        "day": 2
+                    },
+                    "nationality": 9
+                },
+                {
+                    "traveller_id": 614918586,
+                    "type": 0
+                }
+            ],
+            "unit_count": 1,
+            "unit_type": 0
+        },
+        {
+            "query_id": 2,
+            "search_parameters_accommodation": {
+                "location_geo_tree": {
+                    "country": 13,
+                    "region": "Balearic Islands",
+                    "city_or_resort": "Mallorca"
+                }
+            },
+            "travel_period": {
+                "start_date": {
+                    "year": 2024,
+                    "month": 11,
+                    "day": 15
+                },
+                "end_date": {
+                    "year": 2024,
+                    "month": 11,
+                    "day": 18
+                },
+                "flexible": false,
+                "los_from_nights": 894374666,
```

Comment:
Another extreme example of "example data should contain sensible values". This query asks for a length of stay of 2 450 341 years. Good luck finding a hotel which still exists then :D

Reply:
Example messages now have more meaningful values.